.github/workflows/async-signature.yml

```diff
@@ -39,10 +39,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: RustCrypto/actions/cargo-cache@master
-      - uses: dtolnay/rust-toolchain@master
-        with:
-          toolchain: nightly
+      - uses: dtolnay/rust-toolchain@nightly
+      - run: cargo update -Z minimal-versions
+      - uses: dtolnay/rust-toolchain@stable
       - uses: RustCrypto/actions/cargo-hack-install@master
       - run: rm ../../Cargo.toml
```

Comment:
Can't use the shared workflow due to this, FYI (it's nested a level deeper than usual)

Reply:
Maybe it's worth to move `async-signature` to the top level?

Reply:
Yeah, it would help with things like this. There's some open PRs though, so we'd need to get those merged first.